### ISSUE TITLE
Some spells not working

### ISSUE THREAD

**Issue by TheElectricCake** [raphydaphy/EnhancedProgression#7]
Using Vitality 2.0 in the All the Mods mudpack (1.10.2). Made a Tier 2 altar and wand and crafted a few more aggressive spells then put them in my spellbag. The one that shoots arrows works fine, but I cannot use the Tier 2 fireball. It just jerks the wand around with no error message. I also cannot seem to use the Contained Explosion spell. It takes away Essence from my wand but nothing happens.

So far, this mod seems like it could be a lot of fun. The arrows spell certainly is a blast...It's a shame the 'blasting' spells don't seem to work for me.

Have you considered a spell that grants armor or absorption?

It would be nice to have a way to remove spells from the spell bag for testing purposes. 

It would also be nice to have a way to remove items from the altar without breaking the altar in case you accidentally put the wrong thing in. Breaking the altar works, but if you've just put in most of a recipe and got one thing wrong it'd be nice if, say, shift-right-click with an empty hand removes the last object.

**Comment by raphydaphy** [comment]
ah yea sorry :( all fireball spells are currently not working with the spell bag, and the explosion spell is fixed in recent versions (2.2.1 plus) but the version in all the mods is outdated.

There is armor coming soon to the mod, and the Unabridged Immortality spell protects you 100% while u hold rightclick.

Shift+rightclicking the altar to remove items is on my to-do list, ill bump it up higher since alot of people requested it.

And the Spell Forge block can remove spells from the spell bag but its not finished yet so theres no way to make it.

Also in version 2.4 of the mod there is a guidebook that you craft by right-clicking a book with a wand, but that version isnt released yet and will be avalable in 1-4 days.

Basically Vitality 2.0 is a pretty old version, since I've been working super fast on the mod and were already up to 2.4 now. Theres 10+ new spells, a new multitool that mines in an AoE for the cost of essence, new Essence Vials to store bulk amounts of essence and even more new stuff. U should probably manually update it in the pack to the latest if ur using it much since it adds so much new stuff.

Tl;dr, Really great to hear that u like the mod tho, and thanks for the feedback. Shift+rightlicking altar to remove items is for sure coming soon, as well as fireball spell fix, do note that it works when its not in a spell bag tho. Everything else is already fixed in the newer versions of the mod since ur using the outdated version included in All the Mods.
